Project-Lifecycle.md

```diff
@@ -98,6 +98,8 @@ vote to abstain from representation on the TSC.
 
 ## Applying to join
 
+The Node.js Foundation only admits projects written in Node.js.
```

Comment:
I think just "Supporting the Node.js Ecosystem" should be sufficient. Anything else is a poor use of our resources since it's not actually moving _Node_ forward.

Supporting core is also supporting the ecosystem so that doesn't need to be stated.